Modules/ProfileManager/Profile.lua

```diff
@@ -48,6 +58,30 @@ function Profile:ClearMain()
     self.main = ""
 end
 
+function Profile:Alts()
```

Comment:
This function is not used anywhere; it seems to be the only reason for the `altList` to exist; remove this function and the variable...